iceoryx_hoofs/test/stresstests/test_lockfree_queue_stresstest.cpp

```diff
@@ -15,7 +15,7 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-#include "test.hpp"
+#include "iceoryx_hoofs/testing/test.hpp"
```

Comment:
Are those stress tests executed on the CI?

Reply:
Not in this CI since it cannot give us guarantees regarding hardware resources.